pkg/osquery/tables/execparsers/remotectl/parse_test.go

```diff
@@ -0,0 +1,113 @@
+//go:build darwin
+// +build darwin
+
+package remotectl
+
+import (
+	"bytes"
+	"fmt"
+	"os"
+	"path"
+	"regexp"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestParse(t *testing.T) {
+	t.Parallel()
+
+	var tests = []struct {
```

Comment:
It might be worth adding something about the number of expected lines. It's a pretty coarse check, but it can make sure things produce the desired results. 

`validateKeyValueInCommandOutput` looks like it's iterating for correctness, but it's iterating on the result, so if the result has an unexpected number of rows, it'll miss

Reply:
Makes sense, I'll add that!